dotpay.php

```diff
@@ -82,6 +82,8 @@ public function install()
     {
 	
         Configuration::updateValue('DP_TEST', true);
+        Configuration::updateValue('DP_CHK', false);
+        Configuration::updateValue('DP_SSL', false);
```

Comment:
Maybe you could add these configuration keys in an upgrade file, or these configuration values won't exist if the module is already installed..